meson.build

```diff
@@ -442,7 +442,7 @@ endforeach
 libsystemd_sym_path = '@0@/@1@'.format(meson.current_source_dir(), libsystemd_sym)
 libsystemd = shared_library(
         'basu',
-        version : libsystemd_version,
+        soversion : soversion,
```

Comment:
I wonder whether we should use `meson.project_version()` here. I guess if we do a 0.2.0 it won't bump the soversion? Then better to use an explicit soversion I guess.